stream-compositions/cursors/files/sql/transaction_cursor/mssql/create/txn_cursor.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE [dbo].[txn_cursor]
```

Comment:
this is a very simple DDL that could be accomplished by creating a Liquibase changeset, that would be much more aligned with the backbase product direction toward cloud-native applications

Reply:
@uesleilima We're definitely looking for some guidance here. Before the next PR, I'll schedule a meeting with you and some developers to better understand this.